frontend/public/components/pdb/pdb.tsx

```diff
@@ -0,0 +1,272 @@
+import { ListPageCreateLink } from '../factory/ListPage/ListPageCreate';
+import { PodDisruptionBudgetKind, referenceForModel } from '../../module/k8s';
+import VirtualizedTable, { TableData } from '../factory/Table/VirtualizedTable';
+
+import { useActiveColumns } from '../factory/Table/active-columns-hook';
+import {
+  Kebab,
+  ResourceLink,
+  Timestamp,
+  ResourceKebab,
+  ResourceSummary,
+  SectionHeading,
+  navFactory,
+  DetailsItem,
+  Selector,
+} from '../utils';
+import { DetailsPage } from '../factory';
+
+const tableColumnInfo = [
+  { className: '', id: 'name' },
+  { className: '', id: 'namespace' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-sm'), id: 'selector' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-md'), id: 'minAvailable' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-lg'), id: 'disruptionsAllowed' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-xl'), id: 'creationTimestamp' },
+  { className: Kebab.columnClass, id: '' },
+];
+
+const { common } = Kebab.factory;
+const menuActions = [...Kebab.getExtensionsActionsForKind(PodDisruptionBudgetModel), ...common];
+
+const PodDisruptionBudgetTableRow: React.FC<RowProps<PodDisruptionBudgetKind>> = ({
+  obj,
+  activeColumnIDs,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <TableData {...tableColumnInfo[0]} activeColumnIDs={activeColumnIDs}>
+        <ResourceLink
+          kind={referenceForModel(PodDisruptionBudgetModel)}
+          name={obj.metadata.name}
+          namespace={obj.metadata.namespace}
+        />
+      </TableData>
+      <TableData {...tableColumnInfo[1]} activeColumnIDs={activeColumnIDs}>
+        <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
+      </TableData>
+      <TableData {...tableColumnInfo[2]} activeColumnIDs={activeColumnIDs}>
+        <Selector selector={obj.spec.selector} namespace={obj.metadata.namespace} />
+      </TableData>
+      <TableData {...tableColumnInfo[3]} activeColumnIDs={activeColumnIDs}>
+        {_.isNil(obj.spec.maxUnavailable)
+          ? `${t('public~Min available')} ${obj.spec.minAvailable}`
+          : `${t('public~Max unavailable')} ${obj.spec.maxUnavailable}`}
+      </TableData>
+      <TableData {...tableColumnInfo[4]} activeColumnIDs={activeColumnIDs}>
+        {obj.status.disruptionsAllowed}
+      </TableData>
+      <TableData {...tableColumnInfo[5]} activeColumnIDs={activeColumnIDs}>
+        <Timestamp timestamp={obj.metadata.creationTimestamp} />
+      </TableData>
+      <TableData {...tableColumnInfo[6]} activeColumnIDs={activeColumnIDs}>
+        <ResourceKebab actions={menuActions} kind={obj.kind} resource={obj} />
+      </TableData>
+    </>
+  );
+};
+
+const PodDisruptionBudgetList: React.FC<PodDisruptionBudgetsListProps> = (props) => {
+  const { t } = useTranslation();
+
+  const pdbTableColumns = React.useMemo<TableColumn<PodDisruptionBudgetKind>[]>(
```

Comment:
Don't think we need to memo this. Just create a `getPDBTableColumns` outside of this component and use the import of `i18next` directly (instead of `useTranslation`). The main goal here is we want to use i18n in a live fashion. And our export of `i18next` is live-enough for this use-case. As long as the execution of our `get` function is in the React lifecycle (ie in the render function of a component) it will get a live representation of the translation.